src/Pos/Context/Functions.hs

```diff
@@ -4,6 +4,7 @@ module Pos.Context.Functions
        (
          -- * Genesis
          GenesisUtxo(..)
+       , mkGenesisTxpContext
```

Comment:
>Context reexports GenesisUtxo because it was the part of the context
sound like the most strange thing I have ever heard, lets reexprot all datatypes from `Context.hs` then. gromak's motivation sounds more or less reasonable.

`mkGenesisTxpContext` creates the part of context, why not reexprot?